packages/emf/backend/sirius-components-emf-forms/src/main/java/org/eclipse/sirius/components/emf/forms/LocalDateIfDescriptionProvider.java

```
@@ -0,0 +1,149 @@
+
+            if (optionalEObject.isPresent() && optionalEAttribute.isPresent()) {
+                EObject eObject = optionalEObject.get();
+                EAttribute eAttribute = optionalEAttribute.get();
+
+                Object value = eObject.eGet(eAttribute);
+                if (value instanceof LocalDate localDate) {
+                    try {
+                        Instant instant = localDate.atStartOfDay(ZoneId.of("UTC")).toInstant();
+                        return DateTimeFormatter.ISO_INSTANT.format(instant);
+                    } catch (DateTimeException | NullPointerException e) {
+                        // Ignore
+                    }
+                }
+            }
+            return "";
+        };
+    }
+
+    private BiFunction<VariableManager, String, IStatus> getNewValueHandler() {
+        return (variableManager, newValue) -> {
+            IStatus status = new Failure("");
+            var optionalEObject = variableManager.get(VariableManager.SELF, EObject.class);
+            var optionalEAttribute = variableManager.get(EMFFormDescriptionProvider.ESTRUCTURAL_FEATURE, EAttribute.class);
+            if (optionalEObject.isPresent() && optionalEAttribute.isPresent()) {
+                EObject eObject = optionalEObject.get();
+                EAttribute eAttribute = optionalEAttribute.get();
+
+                if (newValue == null || newValue.isBlank()) {
+                    eObject.eSet(eAttribute, null);
+                    status = new Success();
+                } else {
+                    try {
+                        Instant instant = Instant.parse(newValue);
+                        LocalDate localDate = LocalDateTime.ofInstant(instant, ZoneOffset.UTC).toLocalDate();
+                        eObject.eSet(eAttribute, localDate);
+                        status = new Success();
+                    } catch (DateTimeParseException e) {
+                        // Ignore
```

Why ignore the exception?

---

I removed the try catch to be coherent with all others existing providers.
The exception will be catched higher and an error message will be displayed to the user